.travis.yml

```diff
@@ -13,8 +13,9 @@ script:
   - make test
   - bundle exec codeclimate-test-reporter
 env:
-  - RACK_ENV=test
-  - RAILS_ENV=test
+- RACK_ENV=test
+- RAILS_ENV=test
+- COVERAGE=true
```

Comment:
I don't think that was the case. That only applies for matrixed builds and when more than one ENV vars are defined *on the same line*. We were only running one version of Ruby, and we only had one ENV var per line. You can see that only one build was being run here: https://travis-ci.org/18F/identity-dashboard/builds